chroma-common/blockdevices/blockdevice_zfs.py

```diff
@@ -494,3 +540,21 @@ def disable_if_exists(name):
                     break
 
         return error
+
+    @classmethod
+    def terminate_driver(cls, managed_mode):
+        # prune locks owned by THIS process
+        lockfiles = os.listdir(ZfsDevice.ZPOOL_LOCK_DIR)
+
+        for lockfile in lockfiles:
+            with open(ZfsDevice.ZPOOL_LOCK_DIR + lockfile, 'r') as f:
+                contents = f.readlines()
+
+            assert len(contents) == 1 and contents[0].isdigit(), \
```

Comment:
Same code as here: https://github.com/intel-hpdd/intel-manager-for-lustre/blob/693bed79387b7ff2e15164c86ffe4c6c56dea6ba/chroma-common/blockdevices/blockdevice_zfs.py#L117

Should be hoisted to it's own (preferably) function or method